TenancyInformationApi/V1/Factories/TenancyFactory.cs

```diff
@@ -7,12 +7,27 @@ namespace TenancyInformationApi.V1.Factories
 {
     public static class TenancyFactory
     {
-        public static Tenancy ToDomain(this UhTenancyAgreement uhTenancyAgreement, UhAgreementType agreementType)
+        public static Tenancy ToDomain(this UhTenancyAgreement uhTenancyAgreement, UhAgreementType agreementType, UHProperty property = null)
         {
-            return uhTenancyAgreement.ToDomain(agreementType, uhTenancyAgreement.UhTenureType);
+            return uhTenancyAgreement.ToDomain(agreementType, uhTenancyAgreement.UhTenureType, property);
```

Comment:
You could just do the assignments in here?

Reply:
sorry, I'm not sure I understand what you're referring to? Do you mind elaborating